tmt-app/src/Components/QuizStepper.jsx

```diff
@@ -0,0 +1,91 @@
+import * as React from "react";
+import { useLocation } from "react-router-dom";
+import HomeIcon from "../Images/icons/boba_progress/Home.svg";
+import SubjectsIcon from "../Images/icons/boba_progress/Subjects.svg";
+import SoftSkillsIcon from "../Images/icons/boba_progress/SoftSkills.svg";
+import OrderUpIcon from "../Images/icons/boba_progress/OrderUp.svg";
+import {
+  Container,
+  Step,
+  StepConnector,
+  stepConnectorClasses,
+  StepLabel,
+  Stepper,
+  styled,
+} from "@mui/material";
+
+const ProgressConnector = styled(StepConnector)(({ theme }) => ({
+  [`&.${stepConnectorClasses.active}`]: {
+    [`& .${stepConnectorClasses.line}`]: {
+      backgroundColor: theme.palette.Text.main,
```

Comment:
There should be built in text overrides that isn't `Text` with a capital T. FYI we should default variables to lower camel case. pascal casing is typically reserved for components and classes in common JavaScript and React patterns

Reply:
@jwu910 I chose 'Text' because that was the name of the colour preset in Figma, and because it follows the uppercase format of all the other theme.palette variables. Should I refactor all the variable names for our theme.palette then in this commit?

Reply:
we dont _have_ to fix it now if you think it'll be too much. Pointing out the common patterns you'll probably see and can see existing. outside of code (like figma) i wouldn't expect those to follow common coding practices.

If you want, i'd say just open a ticket, and we can fix it later